client/src/pages/Discovery/Discovery.tsx

```diff
@@ -38,13 +43,27 @@ export default function Discovery(): JSX.Element {
       if (allContests.contests) {
         setContests(allContests.contests);
       } else {
-        console.log('No Contests Found');
         new Error('Could Not Get Contests');
       }
     }
     getAll();
   }, []);
 
+  useEffect(() => {
```

Comment:
Please see comment below

Reply:
added changes requested along with the button to reset the filter.